docs/source/cdk/cdk-deployment-personal-aws.rst

```diff
@@ -0,0 +1,45 @@
+Deploy CDK to Personal AWS Account
+==================================
+
+1. Get a personal AWS account setup by submitting a `Service Desk ticket <https://servicedesk.lasp.colorado.edu/servicedesk/customer/portals>`_.
+2. Once the account is setup, `login in to AWS <https://signin.aws.amazon.com/signin?redirect_uri=https%3A%2F%2Fconsole.aws.amazon.com%2Fconsole%2Fhome%3FhashArgs%3D%2523%26isauthcode%3Dtrue%26state%3DhashArgsFromTB_us-east-2_bdb6cea710bddb8a&client_id=arn%3Aaws%3Asignin%3A%3A%3Aconsole%2Fcanvas&forceMobileApp=0&code_challenge=DSG8NACFeAjbOsDQjFFz6TvuW-ohRykiAIXkuEQriOI&code_challenge_method=SHA-256>`_ and create a user in IAM.
+   a. You'll likely want to give your user the `AdministratorAccess` policy.
+3. Next, go to `Security Credentials` for your new user and create an access key. This will be needed in step 6, so save the access key/secret access key.
+4. On your local system, add a new profile to `~/.aws/config` for your personal account. This can be named anything you want, we'll call it `my_profile` for this example.
```

Comment:
```suggestion
4. On your local system, add a new profile to ``~/.aws/config`` for your personal account. This can be named anything you want, we'll call it ``my_profile`` for this example.
```

This is an EXTREME nit-pick but I find that rendering things like this as code syntax (or whatever you call it?) instead of italicized looks nicer in the rendered docs. Not suggesting you need to change this but just wanted to get my opinion out there 😅

Reply:
Thanks for pointing this out! I completely missed this. I think markdown is a single backtick and I'm a lot more used to that than rst.